srcpkgs/jamulus/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'jamulus'
+pkgname=jamulus
+version=3.6.2
+revision=1
+_version=r$(echo ${version} | sed 's/\./_/g')
+wrksrc=${pkgname}-${_version}
+build_style=qmake
+configure_args="Jamulus.pro"
+hostmakedepends="pkg-config"
+makedepends="qt5-declarative-devel jack-devel"
+depends="jack"
```

Comment:
I don't think this should be here; a jack server can be provided by some alternative application, like pipewire.

Reply:
I didn't know about things like pipewire - thanks for the tip! :) removed `depends`.